app/src/org/commcare/activities/components/RoundedButton.java

```diff
@@ -0,0 +1,189 @@
+package org.commcare.activities.components;
+import android.content.Context;
+import android.content.res.TypedArray;
+import android.graphics.Color;
+import android.graphics.Typeface;
+import android.graphics.drawable.Drawable;
+import android.util.AttributeSet;
+import android.view.ViewGroup;
+
+import androidx.annotation.DrawableRes;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.constraintlayout.widget.ConstraintLayout;
+import androidx.core.content.ContextCompat;
+import androidx.core.content.res.ResourcesCompat;
+
+import org.commcare.dalvik.R;
+
+public class RoundedButton extends androidx.appcompat.widget.AppCompatButton {
```

Comment:
What's the reason to have own implementation here, seems like Android have [inbuild support for rounded buttons](https://stackoverflow.com/a/50155799)